tests/python/unittest/test_tir_schedule_blockize.py

```diff
@@ -15,12 +15,10 @@
 # specific language governing permissions and limitations
```

Comment:
A generic question: why do we need to refactor the testing as well? Is it because the new implementation can not pass the current tests?